internal/serverinstall/nomad.go

```diff
@@ -286,19 +255,18 @@ func (i *NomadInstaller) Install(
 	s.Update("Waypoint server ready")
 	s.Done()
 
-	if i.config.consulService {
-		s = sg.Add("The CLI has been configured to automatically install a Consul service for\n" +
+	if i.config.serviceProvider == "consul" && i.config.consulService {
```

Comment:
I recall from a code review meeting that this was supposed to be an "or" instead of an "and"?

Reply:
@xiaolin-ninja since the default values are "consul" and `true` for `serviceProvider` and `consulService`, respectively, we want this to be "and" because if the user sets something other than "consul" for `serviceProvider`, or sets `consulService` to false, we then do not want to create the job with a Consul service.